webaccess/res/simpledesk.js

```diff
@@ -28,22 +28,22 @@ function getPage(uni, page) {
 function connect() {
    var url = "ws://" + window.location.host + "/qlcplusWS";
    websocket = new WebSocket(url);
-   websocket.onopen = function(ev) {
+   websocket.onopen = function() {
     getPage(1, 1);
    };
-   websocket.onclose = function(ev) {
-    console.log(
-      "QLC+ connection is closed. Reconnect will be attempted in 1 second.",
-      e.reason
-    );
+
+   websocket.onclose = function() {
+    console.log("QLC+ connection is closed. Reconnect will be attempted in 1 second.");
```

Comment:
Codacy has a fix for the issue: [Replace `"QLC+·connection·is·closed.·Reconnect·will·be·attempted·in·1·second."` with `⏎······"QLC+·connection·is·closed.·Reconnect·will·be·attempted·in·1·second."⏎····`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9368415)
```suggestion
    console.log(
```